src/openzaak/components/zaken/api/serializers/core.py

```diff
@@ -112,7 +112,8 @@ class Meta:
         extra_kwargs = {
             'url': {
                 'validators': [
-                    ResourceValidator('Zaak', settings.ZRC_API_SPEC, get_auth=get_auth, headers={'Accept-Crs': 'EPSG:4326'})
+                    ResourceValidator('Zaak', settings.ZRC_API_SPEC, get_auth=get_auth,
+                                      headers={'Accept-Crs': 'EPSG:4326'})
```

Comment:
Why is this still a URL?

Reply:
I didn't change the model. It has an URL field to some `Zaak` resource and I thought It is external. 
@sergei-maertens Is it correct?